src/pods/generate/TemplatedResourcesGenerator.ts

```diff
@@ -33,7 +36,7 @@ export class TemplatedResourcesGenerator implements ResourcesGenerator {
    * @param engine - Template engine for generating the resources.
    */
   public constructor(templateFolder: string, factory: FileIdentifierMapperFactory, engine: TemplateEngine) {
-    this.templateFolder = templateFolder;
+    this.templateFolder = resolvePathInput(templateFolder);
```

Comment:
```suggestion
    this.templateFolder = resolveModulePath(templateFolder);
```

Reply:
I saw the name more as "resolve this file path (which may or may not have the $MODULE_PATH variable in it)", hence the name.

Reply:
Yeah, but `Input` is weird. Everything is input.

`resolveAssetPath`?

Reply:
Well it is usable by every class that takes a file path as input. But in practice it is probably only going to be necessary for classes that take a path to an asset as input so this makes sense.